language/tools/move-cli/README.md

```diff
@@ -0,0 +1,197 @@
+# Move CLI
+
+This is a tool for experimenting with Move without validators, a blockchain, or transactions. Persistent data is stored on-disk in a directory structure that mimics the Move memory model
+
+## Installation
+```
+cargo build --release
+export LIBRA_HOME=<path_to_your_libra_dir>
+alias move="$LIBRA_HOME/target/release/move-cli $@"
+```
+
+## Compiling and running scripts
+
+Here is a simple script that prints its `signer`:
+
+```rust
+// TODO: perhaps start with a gentler intro script before going to signers
+script {
+use 0x1::Debug;
+fun main(account: &signer) {
+    Debug::print(account)
+}
+}
+```
+
+Place this in a file named `script.move` and try
+```
+λ move run script.move --signers 0xf
+[debug] (&) { 0000000000000000000000000000000f }
+```
+
+The `--signers 0xf` part indicates which account address(es) have signed off on the script. Omitting `--signers` or passing multiple signers to this single-`signer` script will trigger a type error.
+
+TODO: show script compilation error
+
+TODO: showcase basic arithmetic, control-flow constructs, `Signer` module, `Vector` and `Option` modules.
+
+## Adding new modules via `move_lib`
+
+By default, the CLI compiles and includes the `Compare`, `Debug`, `Errors`, `Option`, `Signer`, and `Vector` modules from the Move standard library. New modules can be added to the `move_lib` directory. The `move run` command will compile and publish each module source file in `move_lib` before running the given script.
```

Comment:
Where does the `move_lib` directory live? I assume anywhere you are creating modules, and not necessarily in the main `libra` repo?

Reply:
Correct; `move_lib` doesn't have to be under `libra`. By default, the CLI looks in the move_lib` subdirectory of wherever you run the binary, and you can point it to a different directory with the `--lib` flag.